content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -274,10 +285,25 @@ A fanout configuration routes traffic from a single IP address to more than one
 based on the HTTP URI being requested. An Ingress allows you to keep the number of load balancers
 down to a minimum. For example, a setup like:
 
-```
-foo.bar.com -> 178.91.123.132 -> / foo    service1:4200
-                                 / bar    service2:8080
-```
+{{< mermaid >}}
+graph LR;
+  client([client])-. Ingress-managed load balancer .->ingress[Ingress, 178.91.123.132];
```

Comment:
```suggestion
  client([client])-. Ingress-managed <br> load balancer .->ingress[Ingress, 178.91.123.132];
```
this, @shuuji3 ?

Reply:
Yes, this should generate a multi-line label with a narrow width.